cmd/geoipupdate/defaults_windows.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"os"
+)
+
+var (
+	// I'm not sure these make sense. However they can be overridden at runtime
+	// and in the configuration, so we have some flexibility.
+	defaultConfigFile        = os.Getenv("SYSTEMDRIVE") + `\ProgramData\MaxMind\GeoIP.conf`
```

Comment:
I think this is _almost_ the best spot. I'd just add a `GeoIPUpdate\` component to the path, such as `os.Getenv("SYSTEMDRIVE") + "\ProgramData\MaxMind\GeoIPUpdate\GeoIP.conf"

Reply:
Makes sense. Thanks!